lmp/bb-config.sh

```diff
@@ -47,6 +48,30 @@ IMAGE_INSTALL_append = " ${EXTRA_IMAGE_INSTALL}"
 EOFEOF
 
 if [ -n "$OTA_LITE_TAG" ] ; then
+	# Ptest-based builds require the same build settings and variables,
+	# but the final image needs to be tagged differently, such as
+	# <main tag>-ptest, so perform the change at the OTA_LITE_TAG variable
+	if [ "${ENABLE_PTEST}" = "1" ]; then
+		IFS=","
+		PTAGS=""
+		for tag in ${OTA_LITE_TAG}; do
+			lmptag=$(echo $tag | cut -d: -f1)
+			PTAGS="${PTAGS} ${lmptag}-ptest"
```

Comment:
It took me a minute to sort out how clever using a "space" was here instead of a comma.